lib/sigs/sigs.go

```diff
@@ -37,9 +37,9 @@ func Verify(sig *crypto.Signature, addr address.Address, msg []byte) error {
 		return xerrors.Errorf("signature is nil")
 	}
 
-	if addr.Protocol() == address.ID {
-		return fmt.Errorf("must resolve ID addresses before using them to verify a signature")
-	}
+	//if addr.Protocol() == address.ID {
```

Comment:
@arajasek The test obviously fails if we keep this(as the ID address is not resolved anywhere during mpool verification). But I have commented this out to show that the test fails even if we don't have this.